wiki/versions/1.19/entity/entity_attributes/groovy/src/main/groovy/org/quiltmc/wiki/entity_attributes/mixin/LivingEntityMixin.groovy

```diff
@@ -0,0 +1,33 @@
+package org.quiltmc.wiki.entity_attributes.mixin
+
+import net.minecraft.entity.LivingEntity
+import net.minecraft.entity.attribute.DefaultAttributeContainer
+import net.minecraft.entity.attribute.EntityAttribute
+import net.minecraft.entity.attribute.EntityAttributeInstance
+import org.quiltmc.wiki.entity_attributes.AttributesExample
+import org.spongepowered.asm.mixin.Mixin
+import org.spongepowered.asm.mixin.Shadow
+import org.spongepowered.asm.mixin.injection.At
+import org.spongepowered.asm.mixin.injection.Inject
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable
+
+@Mixin(LivingEntity.class)
```

Comment:
You're going to need to `@POJO` and `@CompileStatic` this at the *absolute* least for it not to do weird stuff in production; I'd decompile the generated class and check that there's no extra groovy methods/fields in it

Reply:
Yeah, bytecode looks stable with those two. Though it was still 'stable' with dynamic because of the specific requirements needed for a mixin to actually fail.

Reply:
POJO is the more important bit than CompileStatic to not mess stuff up, but yeah